ush/rocoto/setup_expt.py

```diff
@@ -149,6 +161,48 @@ def edit_baseconfig():
     nens = args.nens
     cdump = args.cdump
     gfs_cyc = args.gfs_cyc
+    partition = args.partition
+
+    # Set machine defaults
+    if machine is 'WCOSS_DELL_P3':
```

Comment:
OK, I see now you have most of the parameters defined in the setup python script. It is fine.  Still, users may have to edit these settings locally in their EXPRDIR because not all are using or can use the same disk allocation.

Reply:
The defaults we had in config.base.emc.dyn before were moved into the setup_expt scripts so the final config.base generated will only include the defaults for the machine the user is on. Users will then need to update their config.base before running setup_workflow script, just as they had to before.